src/Util/UnsupportedPropertiesUtil.ts

```diff
@@ -65,7 +65,7 @@ class UnsupportedPropertiesUtil {
     const unsupportedSymbolizerProps = UnsupportedPropertiesUtil
       .getUnsupportedPropsForSymbolizer<T>(unsupportedProperties, symbolizerName);
 
-    const val: SupportDef = unsupportedSymbolizerProps?.[(propName as string)];
+    const val = unsupportedSymbolizerProps?.[(propName as keyof UnsupportedSymbolizerProps<T>)] as SupportDef;
```

Comment:
Can you extract `propName as keyof UnsupportedSymbolizerProps<T>` to a seperate type to enhance readability.

Reply:
See ca81f698.